gateway-service-knoxtoken/src/test/java/org/apache/knox/gateway/service/knoxtoken/TokenServiceResourceTest.java

```diff
@@ -23,6 +23,7 @@
 import com.nimbusds.jose.JWSVerifier;
 import com.nimbusds.jose.crypto.RSASSASigner;
 import com.nimbusds.jose.crypto.RSASSAVerifier;
+import org.apache.commons.lang3.tuple.Pair;
```

Comment:
Java has a Pair class already? Don't think we need commons-lang3.

Reply:
javafx.util.Pair ?

Reply:
Hmm yea, I thought JDK 8 had a "Pair" class. 

Maybe https://docs.oracle.com/javase/8/docs/api/java/util/AbstractMap.SimpleImmutableEntry.html?

JDK 9+ has https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Map.html#entry(K,V)


https://stackoverflow.com/questions/24328679/does-java-se-8-have-pairs-or-tuples

Reply:
commons-lang3 is used in other modules already, and it is only a test dependency in this module.

Reply:
Right but would be better to get rid of it in other modules instead of just using that as an argument to keep adding it. Once it is added as a test module that opens it up to other usages. The only way to prevent usage is to make sure its not included in the pom.

Reply:
I'm just going to replace it with Map.Entry for this test